### DISCUSSION TITLE
Folder compare status missing

### DISCUSSION THREAD

**Discussion by SuneSchrold** [WinMerge/winmerge#672] [General]
When comparing two folders the **Comparison Result** column is blank for folders (see https://winmerge.org/screenshots/?lang=en -> 'Folder Comparison Results'). It looks like the Tree View can actually show if there are differences in folders, as they are marked "Folders are Different" (see https://winmerge.org/screenshots/?lang=en -> 'Folder Compare Tree View').
Is it somehow possible to enable folder comparison result for regular folder comparison?

**Comment by sdottaka** [comment]
An empty "Comparison result" means that the folder was not compared. Perhaps the "Include Subfolders" option is unchecked.

However, if you check the "Include Subfolders" option, folders other than Unique Folders will not be displayed except in tree mode.

So, unfortunately, there is no way to display the "Comparison result" of the folder except in tree mode.
